src/Documentation/FixerDocumentGenerator.php

```diff
@@ -129,7 +130,11 @@ public function generateFixerDocumentation(FixerInterface $fixer): string
 
             $configurationDefinition = $fixer->getConfigurationDefinition();
 
-            foreach ($configurationDefinition->getOptions() as $option) {
+            $options = $configurationDefinition->getOptions();
+
+            usort($options, static fn (FixerOptionInterface $a, FixerOptionInterface $b): int => strcmp($a->getName(), $b->getName()));
```

Comment:
Does that make sense, or shall we keep options sorted in the `FixerConfigurationResolver` instead?

Reply:
Agree. that way, whenever we use options, we would have them sorted - now, we only have it sorted in Docs generator. We use options in other places too, eg `describe` command